src/singletons/Settings.hpp

```diff
@@ -193,8 +193,8 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting streamerModeHideViewerCountAndDuration = {
         "/streamerMode/hideViewerCountAndDuration", false};
     BoolSetting streamerModeMuteMentions = {"/streamerMode/muteMentions", true};
-    BoolSetting streamerModeSupressLiveNotifications = {
-        "/streamerMode/supressLiveNotifications", false};
+    BoolSetting streamerModeSuppressLiveNotifications = {
+        "/streamerMode/suppressLiveNotifications", false};
```

Comment:
Please revert the change on this line as it would revert any users previously set setting